### DISCUSSION TITLE
I don't have email address in college, but have valid id card. What to do, there is no option to add id card. Pls help.

### DISCUSSION THREAD

**Discussion by Anirban-2003** [community/community#146142] [GitHub Education]
### Hi everyone,

I don't have email address in college, but have valid id card. What to do, there is no option to add id card. Pls help.

**Comment by quoc10973** [comment]
You can get by ID card!!

1. Go to the GitHub Education Page
Visit the [GitHub Student Developer Pack](https://education.github.com/pack) page.

2. Start the Verification Process
When you're filling out the form to request access to the pack, you’ll typically be asked for your school-issued email address. Since you don’t have one, proceed to the next steps.

3. Select the Option to Provide Proof of Enrollment
On the verification page, there is an option to provide proof of enrollment, where you can upload your student ID card or other forms of proof. This is where you will submit your valid ID card.

4. Upload Your ID Card
Take a clear photo or scan of your student ID card.
Make sure that your name, the name of your college, and the expiration date (if applicable) are visible and legible on the card.
5. Submit Your Application
After uploading the ID card, submit your application. GitHub Education will review your submission, and if everything looks good, they will grant you access to the Student Developer Pack.

6. Wait for Approval
GitHub Education team usually processes applications within a few days. If your submission is approved, you’ll get access to the benefits.

If You Encounter Issues:
If you’re unsure how to upload your ID card or if the verification process doesn’t give you an option to upload your card, reach out to [GitHub Support](https://support.github.com/contact) and explain your situation. They may be able to assist you directly.
This method should work for you to access the GitHub Education benefits despite not having a school-issued email.

**Reply by Anirban-2003** [reply]
![Screenshot 2024-12-03 232053](https://github.com/user-attachments/assets/7b10a6e2-3d5b-4709-9e59-d367fab393ce)
there is no option to add id card. Can't continue without the edu mail.

**Reply by Chris6077** [reply]
I have the exact same Problem.

I am enrolled at Middlesex University and can provide all documents. However, as remote student I did not get a mail address from them. I got past your window and submitted my documents twice and it got rejected both times because they require that student email for specific universities. Please update me if you find a solution to this. There is no contact info anywhere to get this sorted.

**Reply by skyark21** [reply]
I have the same issue.
